validator/src/main.rs

```diff
@@ -0,0 +1,249 @@
+use std::{net::SocketAddr, process::exit, sync::Arc, time};
+
+use anyhow::{anyhow, bail, ensure, Context, Result};
+use clap::Parser;
+use rand::distributions::{Alphanumeric, DistString};
+use tokio::{
+    io::{AsyncReadExt, AsyncWriteExt},
+    net::{TcpListener, TcpStream},
+    sync::Notify,
+};
+use tracing::{debug, error, info, Instrument};
+
+#[derive(Parser)]
+#[clap(version)]
+struct Args {
+    #[clap(
+        long,
+        env = "CNI_VALIDATOR_LOG_LEVEL",
+        default_value = "cni_validator=info,warn"
+    )]
+    log_level: kubert::LogFilter,
+
+    #[clap(long, env = "CNI_VALIDATOR_LOG_FORMAT", default_value = "plain")]
+    log_format: kubert::LogFormat,
```

Comment:
`CNI_` isn't really appropriate here. `LINKERD_` would be better.

also, the crate isn't called `cni_validator` so this effectively disables all non-warning logs right now

Reply:
Noted, will change both the name of the crate `linkerd_network_validator` and then change the env var for both log level & log format.